src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -54,37 +72,35 @@ export const useAddressAutocomplete = (
     }
   }, [isUSAddress, result.length])
 
-  const fetchSuggestions = useCallback(
-    async ({ search, selected }: { search: string; selected?: string }) => {
-      const params = {
-        key: apiKey,
-        search: search,
+  const fetchSuggestions = useMemo(() => {
```

Comment:
(non-blocking question): I'm trying to wrap my head around the difference between switching from `useCallback` to `useMemo`.

`useCallback` returns a memoized version of the function, while `useMemo` returns a memoized result, right? Why is it preferable to optimize  the result of the throttled function, and not the instance itself? I'm still trying to wrap my head around this. Or is this switch really only to prevent linter errors?

Reply:
Might be answering my own question in realizing that `useMemo()` does memoize a new instance of throttle because that's what is returned as the result.

Reply:
Kind of other way around: `useCallback` returns a memoized version of the function but it will be calling `debounce/throttle` every single time it's called, so you'd get a new copy every time you call it. The calls would then be simply delayed by the specified time rather than throttled or debounced. `useMemo` will memoize it as a _value_ a single time.